### PR TITLE
Add a Knowledge Mining Ontology

### PR DESCRIPTION
**Pull request by JYL61**
Add a Knowledge Mining Ontology

### CODE REVIEW

**Comment by dgarijo** [comment]
@JYL61 please rename .htaccess.txt to `.htaccess` or your redirects will not work

**Comment by JYL61** [comment]
thank you for your information. done~

File: KMO/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Knowledge Mining Ontology (kmo)
+
+Knowledge Mining Ontology (kmo) defines a high-level global schema and provides a vocabulary for users to support automatic knowledge mining through experimental datasets using ontology and ML.
+
+
+# Contacts
+
+*yali Jiang <Jiangy61@cardiff.ac.uk>, @JYL61, @#112478488
```
Comment (TallTed) [inline KMO/README.md:8]:
```suggestion
* yali Jiang <Jiangy61@cardiff.ac.uk>, @JYL61, @#112478488
```

Comment (JYL61) [reply KMO/README.md:8]: thank you~ done

**Pull request merged**